src/client/Client.cc

```diff
@@ -9170,7 +9170,7 @@ int Client::uninline_data(Inode *in, Context *onfinish)
 
 int Client::read(int fd, char *buf, loff_t size, loff_t offset)
 {
-  std::lock_guard lock(client_lock);
+  client_lock.lock();
```

Comment:
Sorry I should have said convert this to `std::unique_lock`

Reply:
I have changed this to std::unique_lock, thanks。